packages/account/src/Sections/Security/Passwords/passwords-platform.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { Button, Icon, Popover, Text } from '@deriv/components';
+import { CFD_PLATFORMS, getPlatformSettings } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import FormSubHeader from 'Components/form-sub-header';
+import SentEmailModal from 'Components/sent-email-modal';
+import { useRequest } from '@deriv/api';
+import { observer, useStore } from '@deriv/stores';
+
+type TPasswordsPlatformProps = {
+    has_dxtrade_accounts?: boolean;
+    has_mt5_accounts?: boolean;
+};
```

Comment:
@likhith-deriv Seems to be prop drilling, We can get these values from client store (or even better, `useFetch`) 🤔

Reply:
There is no prop drilling here. The props are being set from `password` component to be used in `password-platform` component